.github/workflows/cd.yml

```diff
@@ -0,0 +1,38 @@
+name: CD
```

Comment:
Do we really need a CD workflow?

Reply:
We need the CD if we don't want to include the `.github` files in the release I think?